AI/pyneogame/Engine.py

```diff
@@ -148,83 +148,17 @@ def test_player(self, agent):
-
-        # Check that action recieved is a numpy array
-        assert isinstance(agent_action, np.ndarray)
+            # Check that action recieved is a numpy array
+            assert isinstance(agent_action, np.ndarray)
 
-        # Check that the player action produces the required number of cards
-        # Also checks that action can be convertet to boolean array
-        try:
+            # Check that the player action produces the required number of cards
+            # Also checks that action can be convertet to boolean array
+            
             assert len(self.player_hand[agent_action.astype(
-                bool)]) == self.n_cards_to_play
-        except:
-            print('Test failed for ', str(agent))
-            print('Expected number of cards:', self.n_cards_to_play)
-            print('Got: ', len(self.player_hand[agent_action]))
-
-
-def test():
-    """
-    Checking basic functionality of the Game class
-    """
-    game = Game()
-    print('Running Engine tests')
-
-    # Make sure that deal_cards function actually randomizes
-    for _ in range(10):
-        setup_1, setup_2 = game.get_env(), game.deal_cards().get_env()
-        com_arrs = [np.array_equal(a, b) for x, y in zip(
-            setup_1, setup_2) for a, b in zip(x, y)]
-        # There is a chance that arrays will be similar just by chance
-        try:
-            assert sum(com_arrs) < 3
-        except AssertionError:
-            print(com_arrs)
-            print(setup_1)
-            print(setup_2)
-
-    # Test that the scoring function returns expected values
-    assert game.calc_score([1, 1, 1, 1, 1], [2, 3, 4, 0, 0]) == 5
-    assert game.calc_score([2, 3, 4, 0, 0], [1, 1, 1, 1, 1]) == 10
-    assert Game.calc_score([2, 3, 4, 0, 0], [1, 1, 1, 1, 1]) == 10
-    assert Game.calc_score([1, 1, 1, 1, 1], [2, 3, 4, 0, 0]) == 5
-
-    # Check that scoring never exceeds the highest possible score
-    for _ in range(500):
-        arr1 = np.random.randint(0, 5, 4)
-        arr2 = np.random.randint(0, 5, 4)
-        try:
-            max_score = (game.n_cards_on_table + game.n_cards_to_play)**2
-            assert Game.calc_score(arr1, arr2) <= max_score
-        except AssertionError:
-            print(arr1)
-            print(arr2)
-
-    # Scoring should be insensitive to permutations
-    for _ in range(4):
-        arr1 = [1, 1, 4, 4, 2]
-        shuffle(arr1)
-        arr2 = [2, 3, 4, 0, 0]
-        shuffle(arr2)
-        assert Game.calc_score(arr1, arr2) == 7
-        assert Game.calc_score(arr2, arr1) == 6
-
-    # The list of actions should be equal for all game instances
-    for _ in range(10):
-        game1 = Game()
-        game2 = Game()
-        assert (game1.get_actions() == game2.get_actions()).all()
-
-    print('Engine tests completed')
-
-
-def main():
-    test()
-
-
-if __name__ == "__main__":
-    main()
+                    bool)]) == self.n_cards_to_play
```

Comment:
Add the comment as message i.e. assert statement, message

Small typo in the comment.